cmd/tink-worker/cmd/root.go

```diff
@@ -96,6 +86,7 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 				worker.WithLogCapture(captureActionLogs),
 				worker.WithPrivileged(true))
 
+			ctx := context.Background()
 			err = w.ProcessWorkflowActions(ctx)
```

Comment:
Can you use the command context here?
```suggestion
			err = w.ProcessWorkflowActions(cmd.Context())
```

Reply:
This is currently effectively the same thing. Will change if we do `ExecuteContext` in cmd/tink-worker/main.go which I think would give us ctrl-c support? For a later PR though.